src/cli/MeasureUpdateCommand.cpp

```diff
@@ -3,35 +3,6 @@
 *  See also https://openstudio.net/license
 ***********************************************************************************************************************/
 
-/***********************************************************************************************************************
-*  OpenStudio(R), Copyright (c) 2008-2023, Alliance for Sustainable Energy, LLC, and other contributors. All rights reserved.
-*
-*  Redistribution and use in source and binary forms, with or without modification, are permitted provided that the
```

Comment:
and did some manual cleaning up for some files which had the current short license text AND the former one.